packages/ses/src/module-load.js

```diff
@@ -8,6 +8,9 @@
 
 import { create, values, freeze } from './commons.js';
 
+// q, for quoting strings.
+const q = JSON.stringify;
```

Comment:
(Note to self: Gotta move `assert` into SES-shim and get rid of this other definition of `q`)